.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
It's a good idea to add the `.vscode` folder to your `.gitignore` file—you may want to check these settings files in when they're useful to other team members, but mostly not, I think—they might interfere with their _own_ settings, which would be annoying.